src/app/components/NoResults/NoResults.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from "react";
+
+import styled from "styled-components";
+import { Button } from 'tailwind-react-ui'
+import {Link} from 'react-router-dom';
+import notFound from './Assets/noresults.png'
+
+
+const Container = styled.div`
+  width: 80vw;
+  height: 60vh;
+  margin: 1rem;
+  border-radius: 0.7rem;
+  @media screen and (min-width: 768px) and (max-width: 991px) {
+    width: 40vw;
+  }
+  @media screen and (max-width: 768px) {
+    width: 100vw;
+  }
+`;
+
+const ResearchCard1 = (props: any) => {
```

Comment:
Change the component name to something better?

Reply:
is NoResults fine?